src/main/java/core/machines/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.machines;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started work");
```

Comment:
The message 'Bulldozer started work' should be more informative. It's better to indicate that it is a Bulldozer that is working now. For example, 'Bulldozer is now in operation.'